libs/dal/src/repositories/integration/integration.repository.ts

```diff
@@ -60,4 +79,12 @@ export class IntegrationRepository extends BaseRepository<IntegrationEntity> {
       await super.update({ _id: deactivatedIntegration._id }, deactivatedIntegration);
     }
   }
+
+  normalizeQuery(query: FilterQuery<IntegrationEntity & Document>): FilterQuery<IntegrationEntity & Document> {
+    const normalizedQuery = query;
+
+    normalizedQuery.removed = !!normalizedQuery.removed;
```

Comment:
```suggestion
    normalizedQuery.removed = normalizedQuery.removed ?? false;
```
Is this was the intention behind the !!?

Reply:
Yes while converting the result to boolean